docs/examples/partials/activity-filters.html

```diff
@@ -124,19 +124,19 @@
 			ng-if="$ctrl.size !== 'sm'"></tw-select>
 	</div>
 	<div class="col-xs-12 col-sm-6 col-md-3 col-md-pull-3 m-b-2" ng-if="$ctrl.filtersActive">
-		<div class="input-group input-group-sm">
-			<span class="input-group-addon">
-				<label class="control-label">From</label>
-			</span>
-			<input class="form-control" value="01 May 2016" />
-		</div>
+		<tw-date-lookup
+			placeholder="Start date..."
+			ng-model="$ctrl.startDate"
+			ng-max="$ctrl.endDate"
```

Comment:
This is implemented in styleguide components, just using it in the demo page here.  As I mentioned the other day, goal was to be compatible with the native and regular Angular interfaces: https://docs.angularjs.org/api/ng/input/input%5Bdate%5D

I am considering a change to allow different error messages to be used with the dynamic forms, but not quite ready for that.

![image](https://cloud.githubusercontent.com/assets/1318111/26495197/8581bce2-421a-11e7-9c5f-a41a4c77d826.png)